kalite/distributed/features/steps/superuser_create.py

```diff
@@ -46,6 +52,8 @@ def step_impl(context):
 
 @then("the modal won't dismiss")
 def step_impl(context):
+    driver = webdriver.Firefox()
```

Comment:
Also, if you use [from kalite.testing.base_environment import *](https://github.com/learningequality/ka-lite/blob/develop/kalite/testing/base_environment.py) it defines a `before_all` function that adds a WebDriver instance to the context as `context.browser`.